### DISCUSSION TITLE
explode_split  table function 函数错误

### DISCUSSION THREAD

**Discussion by hunan0216** [apache/doris#7628] [A - General / Q&A]
从官网复制下来的语句执行报错    
mysql> select k1, e1 from example1 lateral view explode_split(k2, ',') tmp1 as e1 where k1 = 5 order by k1, e1;
ERROR 1064 (HY000): errCode = 2, detailMessage = Syntax error in line 1:
... e1 from example1 lateral view explode_split(k2, ',') ...
                             ^
Encountered: VIEW
Expected: COMMA

mysql>

**Comment by dahai1996** [comment]
版本问题，0.15不支持 lateral view。1.0.0才支持  https://github.com/apache/incubator-doris/issues/8549

**Comment by hf200012** [comment]
This is only supported in version 1.0
